packages/extension/src/background/index.ts

```diff
@@ -12,6 +11,8 @@ import { BOOT_LOCAL_KEY } from '@dailydotdev/shared/src/contexts/common';
 import { ExtensionMessageType } from '@dailydotdev/shared/src/lib/extension';
 import { getContentScriptPermissionAndRegister } from '../lib/extensionScripts';
 
+const client = new GraphQLClient(graphqlUrl, { fetch: globalThis.fetch });
```

Comment:
Very neat solution. Have you checked whether we can completely avoid having to go through the process below for regular GraphQL queries? Meaning, without using the proxy for the `request` method and the queries would still work?

Reply:
@sshanzel good point, this is only required in the background service worker due to its restricted access to various APIs (such as the `XMLHttpRequest` object). Tested it as well, of course, this works as expected outside of the background worker 🙏

Reply:
Nice, thanks for the verification. The reason I was curious is, that this will remove the overhead of using the `useBackgroundRequest` which complicates the process when making a query/mutation. This means, the requests would automatically work without using those hooks. I understand this is only to migrate the Manifest, so I think we should create a ticket for it to migrate the implementation. Would also make new implementations work automatically as opposed to introducing the said hook every time we do something. cc @rebelchris

Reply:
@DragosIorgulescu we should maybe add a ticket for this. To make implementations less complicated.